Userland/Applications/Piano/WaveWidget.cpp

```diff
@@ -18,7 +18,8 @@ WaveWidget::WaveWidget(TrackManager& track_manager)
 
 int WaveWidget::sample_to_y(int sample) const
 {
-    constexpr int nice_scale_factor = 4;
+    // Sample scaling that looks good, experimentally determined.
+    constexpr double nice_scale_factor = 1.0;
     sample *= nice_scale_factor;
```

Comment:
This doesn't work with a double, `sample` is an `int`. Probably easiest to integrate the calculation into `percentage = ...` below.

Reply:
It compiles though 🤔

Reply:
...but that doesn't mean it performs as expected. This only works because of the 1 value at the moment. If you want to keep this code, please make sure it works for fractional values as well :)